src/ui/logiFlow/components/graphicalUtils.cpp

```
@@ -5,13 +5,24 @@
 GraphicalWireSplitter::GraphicalWireSplitter(QGraphicsItem* parent)
   : GraphicalLogicComponent(std::make_shared<WireSplitter>(Bus(), (std::vector<Bus>){}),
                             nullptr, parent)
+{
+  installSizeCallback();
+};
+
+void GraphicalWireSplitter::installSizeCallback()
 {
   this->associatedComponent->setPropertyCallback("size",
                                                  [this](const PropertyValue& value) {
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)